### PR TITLE
chore/cli-copy-core

### PR DESCRIPTION
**Pull request by carlosallexandre**
## Motivation

- [x] It avoid copying the core to the user's directory when the  `.eventcatalog-core` already exists and has the same version    as the `@eventcatalog/core`.
- [x] It adds a simple logger that wraps Console. Based on the logLevel enabled this logger decides between print or not the message.

### CODE REVIEW

File: bin/logger.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+type LoggerLevel = 'debug' | 'info' | 'warn' | 'error' | 'silent';
```
Comment (boyney123) [inline bin/logger.ts:1]: Hey @carlosallexandre do you think it's worth using https://www.npmjs.com/package/pino for a logging lib? Pretty popular, saves us making our own?

Comment (boyney123) [reply bin/logger.ts:1]: Good question, maybe add into this PR for now, unless there is lots to do, then another PR would be good then remove logging in this one, if that makes sense?

Comment (carlosallexandre) [reply bin/logger.ts:1]: Sure...
I will work on this right now.

**Comment by carlosallexandre** [comment]
Closing in favor #945

**Pull request closed without merging**